images/build/go-runner/cloudbuild.yaml

```diff
@@ -25,7 +25,7 @@ steps:
     - REVISION=$_REVISION
     - GO_VERSION=$_GO_VERSION
     - DISTROLESS_IMAGE=$_DISTROLESS_IMAGE
-    - REGISTRY=$_REGISTRY
+
```

Comment:
nitpick: did you mean to add newline here?

Reply:
to split the section, to make clear, I can revert if you want

Reply:
removed the blank line